src/components/Layout.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { FullWidthContainer, ContentContainer } from './atoms/LayoutContainers';
+
+const Layout: React.SFC<{ children: JSX.Element[] }> = props => {
```

Comment:
If `props` is only `children`, best to just destructure that in the arguments so that there's no confusion as to what you're using: `({ children }) => <FullWidthContainer />`